src/main/java/org/reactome/release/qa/common/AbstractQACheck.java

```diff
@@ -169,4 +171,15 @@ protected boolean isEscaped(GKInstance instance) throws Exception {
         return words;
     }
 
+    /**
+     * Checks contents of skiplist, which should be a list of skippable DbIds, to see if incoming instance should be skipped.
+     * @param inst - GKInstance, instance being checked.
+     * @param skiplistFilename - String, filename of skiplist.
+     * @return - boolean, true if inst DbId is in skiplist, false if not.
+     * @throws IOException, thrown if 'skiplistFilename' doesn't exist as a file.
+     */
+    protected static boolean inSkipList(GKInstance inst, String skiplistFilename) throws IOException {
```

Comment:
In this case, it should be okay to remove the static modifier.  The method should be available to the inheriting QA checks since it is used in non-static methods in the child classes.